src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<DataFromFile> dataList = new ArrayList<>();
```

Comment:
```suggestion
        List<DataFromFile> dataList = new ArrayList<>();
```
use interface type, instead of implementation